### DISCUSSION TITLE
Can i make re-upload old versions on my own profile?

### DISCUSSION THREAD

**Discussion by darthmendorian** [meetric1/gmod-infinite-map#72] [Q&A]
i saw that there was no legacy versions of the infinite map project. luckly i still have some of them left on my hard drive which i want to upload. i already uploaded them but i forgot to ask you Mee for permission. so far nobody has seen it yet so don't worry if you feel like i am trying to steal your project which is not the case at all. this is my github link for the legacy files: https://github.com/bloodyframe2019/Mees-Infinite-Maps-Legacy

**Comment by vurvdev** [comment]
Github and git have commits for a reason. If you want an old version you can just download the repo at a previous commit.

And your repo is not helpful at all considering it doesn't say what commit/version these are from.

Also @Mee12345 you should make releases (there's pre-releases too) so people don't end up asking this, you don't have to attach anything since gh already attaches the source code at the commit of the release

**Comment by DarianLStephens** [comment]
Why is this a thing? As said above, you can just get different versions directly from here.
And besides, it's pretty much just been improvement after improvement. The only thing I'd want, and that I've been porting myself, is the old terrain generation. Much more interesting than flatgrass with some hills.
But with the API, it should be possible for anyone to create a new map which uses that terrain generation no problem.

**Comment by shadow7483147** [comment]
I think the better question to ask is "what license did Mee assign this project?"

**Comment by meetric1** [comment]
yeah, I can make releases probably a good idea

also ur right I probably need a license
